lib/segment/src/types.rs

```diff
@@ -1045,6 +1045,16 @@ pub enum PayloadFieldSchema {
     FieldParams(PayloadSchemaParams),
 }
 
+impl PayloadFieldSchema {
+    pub fn is_numeric(&self) -> bool {
+        matches!(
+            self,
+            PayloadFieldSchema::FieldType(PayloadSchemaType::Integer)
```

Comment:
We probably want an exhaustive match here, otherwise it would be very easy to miss this spot in case we invent another numeric index type

Reply:
Fixed 👍